main_AdamTadele.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+names = []
+
+
+def add_name():
+    print('Press x stop entering a name')
```

Comment:
Can you add more comments and documentation explaining what the function purpose is? Things to consider:
- Function description
- Parameters: data type, parameter description
- Returns: data type, what is returned from the function?